spec/Classes.tex

```diff
@@ -799,48 +923,44 @@ \section{Deleting Class Objects}
 \end{chapeloutput}
 \end{chapelexample}
 
-\begin{openissue}
-Chapel was originally specified without a \chpl{delete} keyword.  The
-intention was that it would support distributed-memory garbage
-collection~(GC).  This is an open research challenge which has been
-put off (potentially indefinitely) in order to focus on more pressing
-issues.  Some effort has also been put into exploring alternatives to
-GC such as support for owned class references and/or borrow checking.
-\end{openissue}
-
-
 \subsection{Class Deinitializer}
 \label{Class_Deinitializer}
 \index{classes!deinitializer}
 \index{deinitializer!classes}
 
-A class author may specify additional actions to be performed before a class object is
-reclaimed, by defining a class deinitializer.  A class deinitializer is a method
-named \chpl{deinit()}.  A class deinitializer takes no arguments (aside from
-the implicit \chpl{this} argument).  If defined, the deinitializer is called each time
-a \chpl{delete} statement is invoked with a valid instance of that class type.  The
-deinitializer is not called if the argument of \chpl{delete} evaluates to \chpl{nil}.
+A class author may create a deinitializer to specify additional actions
+to be performed before memory for instances of that type are reclaimed.
```

Comment:
"before memory for instances of that type are reclaimed" --> "when a class instance is deleted"

I think this way it is more specific.